### DISCUSSION TITLE
How to make a web map of the world

### DISCUSSION THREAD

**Discussion by cliftontoaster-reid** [PrismarineJS/mineflayer#2220] [Q&A]
Hello, I would like to create a project.
It would be an interface where we could access useful information like a map of the area, the players and others.

I was wondering if it was possible to create a map that can be displayed like google map. 

Thank you.

**Comment by Karang** [comment]
You could take inspiration from https://prismarine.js.org/prismarine-rng/?seed=-7621051612768298496#5/0.000/0.000
which is open-source at https://github.com/PrismarineJS/prismarine-rng/tree/master/examples/biomemap
